packages/dashboard-v2/src/components/Tabs/TabPanel.js

```diff
@@ -3,17 +3,17 @@ import PropTypes from "prop-types";
 /**
  * Besides documented props, it accepts all HMTL attributes a `<div>` element does.
  */
-export const TabPanel = ({ children, active, tabId, ...props }) => {
-  if (!active) {
-    return null;
-  }
-
-  return (
-    <div role="tabpanel" id={`tabpanel-${tabId}`} aria-labelledby={`tab-${tabId}`} {...props}>
-      {children}
-    </div>
-  );
-};
+export const TabPanel = ({ children, active, tabId, ...props }) => (
+  <div
+    role="tabpanel"
+    id={`tabpanel-${tabId}`}
+    aria-labelledby={`tab-${tabId}`}
+    {...props}
+    style={{ display: active ? "block" : "none" }}
+  >
+    {children}
+  </div>
+);
```

Comment:
I think the component should return null until it is mounted for the first time (active is true for the first time). That way we don't load the invisible tabs and first paint is faster and we potentially save some unnecessary requests if other invisible tabs would request a list of data or sth like that.

Reply:
Good idea 👍  e3fe7c60